### PR TITLE
[Feature #161046610]add to cart page ui 

### PR DESCRIPTION
**Pull request by eduhmik**
What does this PR do?
Create products and ability to add to Cart.

Description of task to be completed
User should be able to add a product to cart and checkout from the cart page to close a sale

How should this be manually tested?
clone this repo and checkout to ft-cart
products
products

-page-ui-161046415
Open login.html in a browser

Any background context you want to provide?
none

What are the relevant pivotal tracker stories?
#161046415

Screenshots (if appropriate)

### CODE REVIEW

**Comment by jomasim** [comment]
good stuff. Points to note:
-  work on your side bar/side navigation
-  product description text it need to be aligned well 
- Sale-Manager title remove underline use style:  text decoration:none

File: UI/index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                <a href="#">Employee</a>
+        </div>
+        {% endblock %}
+          <!-- Page content -->
+          <div class="main">
+                {% block content %}{% endblock %}
+          </div>
+
+        <div class="footer">
+            {% block footer %}
```
Comment (kwameasiago) [inline UI/index.html:70]: Are you using flask

Comment (eduhmik) [reply UI/index.html:70]: Nope
Just tried using base template didn't work though

**Comment by eduhmik** [comment]
> 
> 
> good stuff. Points to note:
> 
>     * work on your side bar/side navigation
> 
>     * product description text it need to be aligned well
> 
>     * Sale-Manager title remove underline use style:  text decoration:none

Thanks will work on that

**Pull request merged**